app/templates/FAQ.html

```diff
@@ -14,7 +14,14 @@ <h1>Frequency Asked Questions(FAQ)</h1>
         <h4>Learn more about BESST, Powerwalls and Battery Energy<h4>
     </div>
 
+    <div class = "capstone">
+        <div class="intro_vid">
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/jlcd-_b6teo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="border: 2px solid black;"></iframe>
+        </div>
+    </div>
+
     <div class="accordionMenu">
+        <p>Text1</p>
```

Comment:
needs to be replaced with the proper text - there are two more tags like this below